_schemaV2.graphql

```diff
@@ -6527,6 +6533,11 @@ enum sort {
   ASC
 }
 
+type StaticContent {
```

Comment:
`StaticContent` for v2 should also expose `id`, `internalID`, and `slug`. One sec and a I can point to an example of that.